src/crimson/os/seastore/segment_manager/ephemeral.cc

```diff
@@ -34,7 +34,7 @@ device_config_t get_ephemeral_device_config(
 {
   assert(num_devices > index);
   magic_t magic = 0xabcd;
-  auto type = device_type_t::SEGMENTED;
+  auto type = device_type_t::SSD;
```

Comment:
Might need to distinguish the ephemeral device.

How about `device_type_t::SEGMENTED_EPHEMERAL` ?

Reply:
done.

Is it necessary to add another `device_type_t::RANDOM_BLOCK_EPHEMERAL`?

Reply:
Yeah, for the same reason.